modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Specifying AWS Subnets for OpenShift Ingress Load Balancers at Installation
+
+Perform the following steps to allocate API and Ingress load balancers to specific subnets.
+
+.Prerequisites
+
+Before you begin, ensure you have:
+
+* An existing AWS virtual private cloud (VPC).
+
+* Pre-configured AWS subnets intended for use by the OpenShift cluster, with the following considerations:
+
+** A list of their subnet IDs (for example, `subnet-0123456789abcdef0`). These IDs will be used in the `install-config.yaml` file.
+
+** It is highly recommended that you use subnets spanning at least two availability zones (AZs) for high availability of load balancers and other critical components like control planes.
+
+** Sufficient available IP addresses within these subnets for all assigned roles.
+
+** The AWS configuration for these subnets, including Network ACLs and Security Groups, must permit necessary traffic for all roles assigned to them. For subnets hosting an Ingress Controller, this typically includes TCP ports 80 and 443 from desired sources.
+
+* The OpenShift installer binary for your target OpenShift version.
+
+* An `install-config.yaml` file.
+
+.Procedure
+
+. Prepare the `install-config.yaml` file:
++
+If you haven't already, generate the installation configuration file using the OpenShift installer:
++
+[source,terminal]
+----
+$ openshift-install create install-config --dir=<your_installation_directory>
+----
++
+This command creates the `install-config.yaml` file in the specified directory.
+
+. Define Subnets and assign roles:
++
+Open the `install-config.yaml` file located in `<your_installation_directory>` using a text editor. You will define your VPC subnets and their designated roles under the `platform.aws.vpc.subnets` field.
++
+For each AWS subnet you intend the cluster to use, you will create an entry specifying its `id` and a list of `roles`. Each role is an object with a `type` key. To designate a subnet for the default Ingress Controller, assign it a role with `type: IngressControllerLB`.
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+metadata:
+  name: my-cluster # Example cluster name
+platform:
+  aws:
+    region: us-east-1 <2>
+    vpc: <3>
+      subnets: <4>
+      - id: subnet-0fcf8e0392f0910d5 # Public Subnet in AZ us-east-1a <5>
+        roles:
+        - type: IngressControllerLB <6>
+        - type: Bootstrap
```

Comment:
Should be `BootstrapNode`

Reply:
+1 Slightly different from EP. Reference [here](https://github.com/openshift/installer/blob/321604f6d9e1102f50450b9bf5fb1b853c0acb52/pkg/types/aws/platform.go#L221) :D